e2e/app/home-page.ts

```diff
@@ -32,7 +32,7 @@ export default class HomePage extends AuthenticatedPage {
   }
 
   async getCreateNewWorkspaceLink(): Promise<ElementHandle> {
-    return findIcon(this.page, FIELD_LABEL.CREATE_NEW_WORKSPACE, 'plus-circle');
+    return findIcon(this.page, {text: FIELD_LABEL.CREATE_NEW_WORKSPACE}, 'plus-circle');
```

Comment:
nit: could we make an enum  with these icon type strings?

Reply:
good suggestion. I can refactor code in a new PR.